live-examples/js-examples/intl-collator-supportedlocalesof.html

```diff
@@ -0,0 +1,8 @@
+<pre>
+<code id="static-js">var locales1 = ['ban', 'id-u-co-pinyin', 'de-ID'];
+var options1 = { localeMatcher: 'lookup' };
+
+console.log(Intl.Collator.supportedLocalesOf(locales1, options1));
+// expected output: Array ["id-u-co-pinyin", "de-ID"]
```

Comment:
I'm not seeing any difference between browsers - but I'll add a comment just in case.

Reply:
I don't mean that the output necessarily if different, but that it could be: as I read it, two browsers could both be spec-compliant but return different things here. It's a really minor point if they don't actually differ, I suppose, but maybe worth noting.

Reply:
👍